packages/core/src/App/Containers/RealAccountSignup/terms-of-use.jsx

```diff
@@ -11,8 +11,7 @@ import {
 import { isDesktop, isMobile } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
 import CheckboxField from 'App/Containers/RealAccountSignup/checkbox-field.jsx';
-import { Hr } from './currency-selector.jsx';
-import { SharedMessage, BrokerSpecificMessage } from './terms-of-use-messages.jsx';
+import { SharedMessage, BrokerSpecificMessage, Hr } from './terms-of-use-messages.jsx';
```

Comment:
I think we can declare `Hr` here as a helper component and not in **terms-of-use-messages.jsx** because it's not a message.

Reply:
Ah ignore this as we're using `Hr` there too. Fine.